scripts/install/install.py

```diff
@@ -314,9 +314,7 @@ def get_rc_file_path():
     rc_file = rc_file or prompt_input_with_default('Enter a path to an rc file to update', default_rc_file)
     if rc_file:
         rc_file_path = os.path.realpath(os.path.expanduser(rc_file))
-        if os.path.isfile(rc_file_path):
-            return rc_file_path
-        print_status("The file '{}' could not be found.".format(rc_file_path))
+        return rc_file_path
```

Comment:
@jtpereyda for the use case you described, shouldn't you be creating the file in here?

I know you mentioned all of the file not found exceptions would be caught later which I agree looks to be true, but if we never create the file then it seems like it doesn't solve your issue.

Reply:
@mross See `_modify_rc` and particularly line 305 were the file is opened:
```python
        with open(rc_file_path, 'a') as rc_file:
            rc_file.write('\n' + line_to_add + '\n')
```

Files opened this way are created if they do not already exist. Sample command line session:
```bash
> cat hello
cat: hello: No such file or directory
> python
Python 2.7.14 (v2.7.14:84471935ed, Sep 16 2017, 12:01:12)
[GCC 4.2.1 (Apple Inc. build 5666) (dot 3)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> with open('hello', 'a') as f:
...   f.write('hi\n')
...
>>> ^D
> cat hello
hi
```

I guess the directory not existing is another error case which I haven't tested.